astropy/table/tests/test_mixin.py

```diff
@@ -131,15 +131,73 @@ def test_io_quantity_write(tmpdir):
         assert 'cannot write table with mixin column(s)' in str(err.value)
 
 
+@pytest.mark.parametrize('table_types', (Table, QTable))
+def test_io_time_write(tmpdir, table_types):
+    """
+    Test that table with Time mixin column can be written by io.fits but
+    not by io.votable and io.misc.hdf5. Validation of the output is done.
+    Test that io.fits writes a table containing Time mixin columns that can
+    be considerably round-tripped (metadata scale, format).
+    """
+    t = table_types([[1,2], ['string', 'column']])
+    for scale in time.TIME_SCALES:
+        t['a'+scale] = time.Time([1,2], format='cxcsec', scale='tai',
+                                  location=EarthLocation(-2446353.80003635,
+                                  4237209.07495215, 4077985.57220038, unit='m'))
+        t['b'+scale] = time.Time(['1999-01-01T00:00:00.123456789', 
+                                  '2010-01-01T00:00:00'], format='isot', scale='utc')
+    t['c'] = [3., 4.]
+
+    filename = tmpdir.join("table-tmp").strpath
+    open(filename, 'w').close()
```

Comment:
Oh yes. I changed that in test_io_quantity_write but forgot to do it here. Will do so.